### ISSUE TITLE
join some no labeled image to train dataset

### ISSUE THREAD

**Issue by duoduo1234** [HiKapok/SSD.TensorFlow#97]
i wanted to join some no labeled image to train dataset,such as neg samples,but didn't success.can someone have  advice?Thank you!

**Comment by jamiechoi1995** [comment]
@duoduo1234 do you mean to add dont care region which does not contribute to the loss?

**Comment by duoduo1234** [comment]
no,closed
